string req
---
bool res
int32 markerid 
